deploy/helm/wg-access-server/templates/ingress.yaml

```diff
@@ -29,6 +29,6 @@ spec:
           - path: /
             backend:
               serviceName: {{ $fullName }}-web
-              servicePort: 80
+              servicePort: 8000
```

Comment:
Hey, not too sure about this one. The service is configured with port 80 [here](https://github.com/Place1/wg-access-server/blob/master/deploy/helm/wg-access-server/templates/service.yaml#L11)

Reply:
Yea I tested it like 5 times to double check. I've always done 1:1 mapping across the board of port and service.

But it looks like ingress needs the container's port, not the service port.

.... *goes to read*

Okay, https://stackoverflow.com/questions/49829452/why-ingress-serviceport-can-be-port-and-targetport-of-service seems so very confusingly confirm that. It points to the endpoint not the service.

```
kubectl get endpoints -n vpn
NAME                         ENDPOINTS            AGE
wg-access-server-web         10.244.1.252:8000    3h59m
wg-access-server-wireguard   10.244.1.252:51820   3h59m
```

Reply:
that's surprising because i'm running in k8s right now with the current helm chart and it works.

don't worry, i'll test this one as well when I merge it (hopefully over the weekend).

Reply:
AHHAAAAAAAAAAA

Okay, i just tested it again (nginx-ingress)

80 works fine
8000 works fine
1234 gives me `503 Service Temporarily Unavailable nginx/1.17.8`

So i'm very very confused why it failed in the past. I'll revert the change.